src/test/resources/Data.json

```diff
@@ -7,5 +7,10 @@
   "links": "Facebook,Twitter,YouTube",
   "placeholders": "Search Facebook,f,Search",
   "policyElementsWhichNeedToClick": "Return Policy,Terms Of Use,Security,Privacy,Sitemap,EPR Compliance",
-  "policyHeaders": "Returns Policy,Flipkart Terms of Use,Safe and Secure Shopping,Privacy Policy,Sitemap,FLIPKART INDIA PRIVATE LIMITED E-WASTE RECYCLING POLICY"
+  "policyHeaders": "Returns Policy,Flipkart Terms of Use,Safe and Secure Shopping,Privacy Policy,Sitemap,FLIPKART INDIA PRIVATE LIMITED E-WASTE RECYCLING POLICY",
+  "headerName": "AppKey",
+  "headerValue":"qP2wzibM0y9rLeRc3jAZQEoBMGgtVGj7",
+  "contentName": "Content-Type",
+  "typeName": "application/json",
```

Comment:
Make the naming more specifc :
AppKeyHeader
AppKeyValue
ContentTypeHeader
ContentTypeValue